cdi/4.0/_index.md

```diff
@@ -6,6 +6,22 @@ summary: "Release for Jakarta EE 10"
 
 Jakarta Contexts Dependency Injection specifies a means for obtaining objects in such a way as to maximize reusability, testability and maintainability compared to traditional approaches such as constructors, factories, and service locators (e.g., JNDI).
 
+The Jakarta Contexts and Dependency Injection 4.0 goals for this release are:
+
+* Split the existing 3.0 specification into pieces that separate out requirements based on the target runtime container support; Java SE, Jakarta Servlet, Jakarta Message Service, etc.
```

Comment:
The main thing is to define CDI Lite portion from CDI Full, which enables build time compilation support.